TACC/Core-Portal
# bug/FP-1567 - Toolbar state not refreshing when moving to shared workspaces ProjectsList

## Overview: ##

## Related Jira tickets: ##

* [FP-1567](https://jira.tacc.utexas.edu/browse/FP-1567)

## Summary of Changes: ##
- Added a bool to filePermissions util to check for this case and ensure toolbar actions are disabled whenever user navigates to Shared Workspaces (not within a project's files listing). (See notes for rationale)

## Testing Steps: ##
1. Go to any storage system under Data Files
2. Select at least one file (**and do not perform any toolbar actions**)
3. Navigate to Shared Workspaces
4. Verify that no toolbar actions are enabled

## UI Photos:

## Notes: ##
I felt this was the simplest approach to this bug which feels like an edge case. I know this doesn't solve the root issue (that the toolbar state is not refreshing), but as users will be in the projects listing only to navigate to a files listing which will then cause a toolbar state refresh, doing anything more complicated felt unnecessary.

My other idea was to add a new watch function in the datafiles saga to force a state refresh, but generating an API call for this felt overcomplicated.
